src/Symfony/Component/AssetMapper/Command/ImportMapOutdatedCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Command;
+
+use Symfony\Component\AssetMapper\ImportMap\ImportMapUpdateChecker;
+use Symfony\Component\AssetMapper\ImportMap\PackageUpdateInfo;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(name: 'importmap:outdated', description: 'Lists outdated JavaScript packages and their latest versions')]
+final class ImportMapOutdatedCommand extends Command
+{
+    private const COLOR_MAPPING = [
+        'update-possible' => 'yellow',
+        'semver-safe-update' => 'red',
+    ];
+
+    public function __construct(
+        protected readonly ImportMapUpdateChecker $updateChecker,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument(
+                name: 'packages',
+                mode: InputArgument::IS_ARRAY | InputArgument::OPTIONAL,
+                description: 'List packages\' names',
+            )
+            ->addOption(
+                name: 'format',
+                mode: InputOption::VALUE_REQUIRED,
+                description: sprintf('The output format ("%s")', implode(', ', $this->getAvailableFormatOptions())),
+                default: 'txt',
+            )
+            ->setHelp(<<<'EOT'
+The <info>%command.name%</info> command will list the latest updates available for the 3rd party packages in <comment>importmap.php</comment>.
+Versions showing in <fg=red>red</> are semver compatible versions and you should upgrading.
+Versions showing in <fg=yellow>yellow</> are major updates that include backward compatibility breaks according to semver.
+
+   <info>php %command.full_name%</info>
+
+Or specific packages only:
+
+   <info>php %command.full_name% <packages></info>
+EOT
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $packages = $input->getArgument('packages');
+        $packagesUpdateInfos = $this->updateChecker->getAvailableUpdates($packages);
```

Comment:
Should this be used in the update command too ?

Reply:
Can you explain more? Do you mean: in the `update` command, we would first use this function to determine *which* updates are possible and then loop over and make them? If so, that's a different way than we're doing it now - but I can't think of any big difference in final behavior.

Reply:
Yes that was the idea, but i'm not talking about "right now" ... but in a near future, as we already said :)